### PR TITLE
Implemented: Added Static UI for Open Orders screen (#1vju63j)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
Internationalization and splitpane menu is not handled in this PR. It will be done in seperate PRs.

**Comment by azkyakhan** [comment]
Desktop-open-orders

**Comment by azkyakhan** [comment]
Mobile-open-orders

**Comment by azkyakhan** [comment]
Picklist-size-filter

**Comment by azkyakhan** [comment]
Assign-picker-modal

File: src/views/AssignPickerModal.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        </ion-button>
+      </ion-buttons>
+      <ion-title>Assign Pickers</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+ <ion-content class="ion-padding">
+   <ion-searchbar />  
+   <div class="filters">
+     <ion-row class="filter-scroll">
```
Comment (dt2patel) [inline src/views/AssignPickerModal.vue:16]: you have already added display flex to filters, you don't need ion-row

File: src/theme/variables.css [hunk 2]
```diff
@@ -233,4 +233,23 @@ http://ionicframework.com/docs/theming/ */
...
     --ion-card-background: #1e1e1e;
   }
+}
+
+.filters {
+  display: flex;
+  overflow: scroll;
+}
+
+.filter-scroll {
```
Comment (dt2patel) [inline src/theme/variables.css:243]: unnecessary class.

File: src/views/AssignPickerModal.vue [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>Assign Pickers</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+ <ion-content class="ion-padding">
```
Comment (dt2patel) [inline src/views/AssignPickerModal.vue:13]: I think we can remove ion-padding here.

File: src/views/OpenOrders.vue [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+    
+    <ion-content :fullscreen="true">
+      <ion-searchbar /> 
+
+      <div class="filters">
+        <ion-row class="filter-scroll">
+          <ion-item lines="none">
+            <ion-checkbox slot="start"/>
+            <ion-label>
+              <h5>Same day</h5>
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:22]: just use ion label here. why are you using h5?

File: src/views/OpenOrders.vue [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+            <ion-checkbox slot="start"/>
+            <ion-label>
+              <h5>Loyalty</h5>
+              <p>37 orders, 40 items</p>
+            </ion-label>
+          </ion-item>
+        </ion-row>
+      </div> 
+
+      <ion-button class="desktop-print" fill="outline" @click="assignPickers">Print Picksheet</ion-button>
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:43]: change class name to desktop-only, make it a generic utility class and set it globally

File: src/views/OpenOrders.vue [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+              <p>Ordered 27th January 2020 9:24 PM EST</p>
+            </ion-label>
+          </div>
+
+          <div class="order-tags">
+            <ion-chip outline>
+              <ion-icon :icon="pricetag" />
+              <ion-label>NN10584</ion-label>
+            </ion-chip>
+            <ion-button fill="clear" class="mobile-recycle" color="danger">
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:59]: same as desktop only class, make it a generic utility class in global file.

File: src/views/OpenOrders.vue [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+        </div>
+
+        <div class="order-item">
+          <div class="product-info">
+            <ion-item lines="none">
+              <ion-thumbnail>
+                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
+              </ion-thumbnail>
+              <ion-label>
+                <h6>WJ06-XL-PURPLE</h6>
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:79]: this should all be in p tag with overline class.

File: src/views/OpenOrders.vue [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+                <p>Blue XL</p>
+              </ion-label>
+            </ion-item>
+          </div>
+          <div class="product-metadata">
+            <ion-note>49 pieces in stock</ion-note>
+          </div>
+        </div>
+
+        <div class="positive-action"></div>
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:90]: need to create actions div that wraps positive and negative actions

File: src/views/OpenOrders.vue [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+
+        <div class="negative-action">
+          <ion-button fill="outline" color="danger">Recycle</ion-button>
+        </div>
+      </ion-card>
+
+      <ion-fab class="mobile-print" vertical="bottom" horizontal="end" slot="fixed">
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:97]: update mobile only utility class name.

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/App.vue [hunk 10]
```diff
@@ -1,11 +1,38 @@
 <template>
   <ion-app>
-    <ion-router-outlet />
+    <ion-menu side="end" type="overlay" menu-id="filter" content-id="main">
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/App.vue]: Done

File: src/views/AssignPickerModal.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/AssignPickerModal.vue]: Done

File: src/theme/variables.css (hunk 2 above)
Comment (azkyakhan) [reply src/theme/variables.css]: Removed sir

File: src/views/AssignPickerModal.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/AssignPickerModal.vue]: Done

File: src/views/OpenOrders.vue (hunk 4 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue]: Removed sir

File: src/views/OpenOrders.vue (hunk 5 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue]: Done

File: src/views/OpenOrders.vue (hunk 6 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue]: Done sir

File: src/views/OpenOrders.vue (hunk 7 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue]: Done

File: src/views/OpenOrders.vue (hunk 8 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue]: Done sir

File: src/views/OpenOrders.vue (hunk 9 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue]: Done sir

File: src/views/OpenOrders.vue [hunk 11]
```diff
@@ -0,0 +1,220 @@
...
+      print,
+      refreshCircleOutline
+    }
+  }
+});
+</script>
+
+<style scoped>
+
+.filters > ion-item {
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:155]: this should be a global class since we are adding this for all filters.

File: src/views/OpenOrders.vue [hunk 12]
```diff
@@ -0,0 +1,220 @@
...
+ flex: 1 0 100%;
+ max-width: 200px;
+ border: 0.01px solid black;
+ border-radius: 10px;
+}
+
+.card-header {
+  display: grid;
+  grid: "tags"
+        "current"
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:165]: why are these named current and next?

File: src/views/OpenOrders.vue [hunk 13]
```diff
@@ -0,0 +1,220 @@
...
+  grid-area: next;
+}  
+
+.order-item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}  
+
+ion-thumbnail> img {
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:192]: global class to use on all pages for all images

File: src/views/OpenOrders.vue [hunk 14]
```diff
@@ -0,0 +1,220 @@
...
+  object-fit: contain;
+}
+
+@media (min-width: 991px) {
+  .card-header {
+    grid: "current tags next" / max-content 1fr max-content;
+    justify-items: center;
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:199]: why are we justifying items to the center in desktop?

File: src/views/OpenOrders.vue [hunk 15]
```diff
@@ -0,0 +1,220 @@
...
+
+  .order-metadata {
+    text-align: end;
+  }
+
+  .order-item {
+    border-bottom: 1px solid black;
+  }  
+
+  .desktop-recycle {
```
Comment (dt2patel) [inline src/views/OpenOrders.vue:215]: where is this class being used?

**Review by dt2patel — CHANGES_REQUESTED** [review]
Please make suggested changes.

File: src/views/OpenOrders.vue (hunk 15 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue:215]: Removed

File: src/views/OpenOrders.vue (hunk 13 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue:192]: Done

File: src/views/OpenOrders.vue (hunk 12 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue:165]: Updated area name

File: src/views/OpenOrders.vue (hunk 11 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue:155]: Done

File: src/theme/variables.css [hunk 16]
```diff
@@ -233,4 +233,42 @@ http://ionicframework.com/docs/theming/ */
...
+  display: none;
+}
+
+.mobile-only {
+  display: unset;
+}
+
+@media  (min-width: 991px){
+  .desktop-only {
+    display: block;
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/theme/variables.css:279]: > when reverting back to visible display type we should not define exact display type.

If we are just setting display unset and not defining display type, button is expanding all over the page, that is why we have set display type to block.

File: src/views/AssignPickerModal.vue [hunk 17]
```diff
@@ -0,0 +1,90 @@
...
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>Assign Pickers</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+ <ion-content>
+   <ion-searchbar />  
+   <div class="filters">
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/views/AssignPickerModal.vue]: Done

File: src/views/AssignPickerModal.vue [hunk 18]
```diff
@@ -0,0 +1,90 @@
...
+  setup() {
+    return {
+      close
+    };
+  },
+});
+</script>
+
+<style scoped>
+.filters > ion-chip {
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/views/AssignPickerModal.vue]: Done

File: src/views/OpenOrders.vue (hunk 14 above)
Comment (azkyakhan) [reply src/views/OpenOrders.vue]: Used justify self on child level.

**Review by dt2patel — APPROVED** [review]
looks good

**Pull request merged**